doc_src/index.hdr.in

```diff
@@ -1068,16 +1070,15 @@ Command mode is also known as normal mode.
 
 - @key{[} and @key{]} search the command history for the previous/next token containing the token under the cursor before the search was started. See the <a href='#history'>history</a> section for more information on history searching.
 
-- @key{Control, X} copies the current buffer to the system's clipboard, @key{Control, V} inserts the clipboard contents.
 - @key{Control,C} deletes the entire line.
 
 - @key{Backspace} moves the cursor left.
 
 \subsubsection vi-mode-insert Insert mode
 
-- @key{Escape} or @key{Control,C} enters <a href="#vi-mode-command">command mode</a>.
+- @key{Escape} enters <a href="#vi-mode-command">command mode</a>.
 
-- @key{Control,x} moves the cursor to the end of the line. If an autosuggestion is available, it will be accepted completely.
+- @key{Control,C} deletes the entire line.
```

Comment:
I just copied the description from the Shared Bindings section. Given that `Control+C` is a shared binding, do you think it would be a good idea to just delete this binding from the vim insert mode section?